src/btree/bt_sync_obsolete.c

```diff
@@ -429,8 +455,13 @@ __checkpoint_cleanup_walk_btree(WT_SESSION_IMPL *session, WT_ITEM *uri)
     while ((ret = __wt_tree_walk_custom_skip(
               session, &ref, __checkpoint_cleanup_page_skip, NULL, flags)) == 0 &&
       ref != NULL) {
-        if (F_ISSET(ref, WT_REF_FLAG_INTERNAL))
+        if (F_ISSET(ref, WT_REF_FLAG_INTERNAL)) {
```

Comment:
nit: no need for {}

Reply:
Because of the macro `WT_WITH_PAGE_INDEX` we need the {}.